src/Statement.php

```diff
@@ -209,6 +207,74 @@ public function sign($privateKeyFile, $privateKeyPass, $options = array()) {
         $this->addAttachment($attachment);
     }
 
+    public static function validateNameOfRSAlgorithm($algorithm)
```

Comment:
Can't we flip the assertion and it is 3 line code block? If anything I'd pull the algorithm name list out to make it more obvious.